config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
-  get 'static_pages/home'
-  get 'static_pages/help'
-  get 'static_pages/about'
   root 'static_pages#home'
+  get '/help',    to: 'static_pages#help'
+  get '/about',   to: 'static_pages#about'
+  get '/contact', to: 'static_pages#contact'
+  get '/signup',  to: 'users#new'
```

Comment:
- Bạn thay đổi `single quote `=> `double quote` trong file này nhé
- Thừa khoảng trắng: `get "/help", to: "static_pages#help"` (dòng 4-6 fix tương tự nhé)